lib/did-veres-one.js

```diff
@@ -302,6 +305,9 @@ api.attachInvocationProof = util.callbackify(async function(
       capabilityAction
     }
   });
+  return {
+    operation: op
```

Comment:
switch to:
```
const operation = ....

...

return { operation };
```

Reply:
`return {operation};`

Use our linter :)